backend/auth/auth.go

```diff
@@ -0,0 +1,75 @@
+package auth
+
+import (
+	"fmt"
+	"github.com/MicahParks/keyfunc"
+	"github.com/ZPI-2024-25/KubernetesAccessManager/models"
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/joho/godotenv"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+)
+
+var jwks *keyfunc.JWKS
+
+func init() {
+	err := godotenv.Load()
+	jwksURL := os.Getenv("KEYCLOACK_URL")
```

Comment:
I don't think adding deafult value is a good idea, because path can change very much depending on installation process. If it's empty we get error
![image](https://github.com/user-attachments/assets/ac9ad8d6-7509-4c04-809a-10cc056a3b85)
But it's not very suggestive, so I will add some messafe

Reply:
Yeah, you are right. Default value doesn't make much sense.

Reply:
Done